Makefile

```diff
@@ -493,37 +492,41 @@ docker-push-base-images:
 	@echo "RESTART ANY DEV SHELLS to make sure they use your new images."
 
 docker-update-base:
-	$(MAKE) docker-base-images go/apis/envoy
+	$(MAKE) docker-base-images pkg/apis/envoy
 	$(MAKE) docker-push-base-images
 
 ambassador-docker-image: ambassador.docker
-ambassador.docker: Dockerfile base-go.docker base-py.docker $(ENVOY_FILE) $(WATT) $(KUBESTATUS) $(WRITE_IFCHANGED) ambassador/ambassador/VERSION.py FORCE
+ambassador.docker: Dockerfile base-go.docker base-py.docker $(ENVOY_FILE) $(WATT) $(KUBESTATUS) $(WRITE_IFCHANGED) python/ambassador/VERSION.py FORCE
 	docker build --build-arg ENVOY_FILE=$(ENVOY_FILE) --build-arg BASE_GO_IMAGE=$(BASE_GO_IMAGE) --build-arg BASE_PY_IMAGE=$(BASE_PY_IMAGE) $(DOCKER_OPTS) -t $(AMBASSADOR_DOCKER_IMAGE) .
 	@docker image inspect $(AMBASSADOR_DOCKER_IMAGE) --format='{{.Id}}' | $(WRITE_IFCHANGED) $@
 
-kat-client-docker-image: kat-client.docker
-.PHONY: kat-client-docker-image
-kat-client.docker: kat-client-docker-image/Dockerfile base-py.docker kat-client-docker-image/teleproxy kat-client-docker-image/kat_client $(WRITE_IFCHANGED) $(var.)KAT_CLIENT_DOCKER_IMAGE
-	docker build --build-arg BASE_PY_IMAGE=$(BASE_PY_IMAGE) $(DOCKER_OPTS) -t $(KAT_CLIENT_DOCKER_IMAGE) kat-client-docker-image
+build/kat/client: kat-client.docker
+.PHONY: build/kat/client
```

Comment:
This is a little weird that `build/THING` is being used both for real files and for `.PHONY` targets.

Reply:
What would you recommend? `build/kat/client.docker` everywhere instead of `build/kat/client`? Just using `kat-client` as a `.PHONY`? 

I can see how it's a little weird, I'm just not sure what's better here...

Reply:
Yes, I forgot to remove the `.PHONY` targets. Would that be ok?